content/statefulset/storageclass.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Define Storageclass"
+
+date: 2018-08-07T08:30:11-07:00
+weight: 5
+---
+#### Introduction
+[Dynamic Volume Provisioning](http://localhost:1313/statefulset/storageclass/) allows storage volumes to be created on-demand. [StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) should be pre-created to define which provisoner should be used and what parameters should be passed when dynamic provisioning is invoked.
+(See parameters for [AWS EBS](https://kubernetes.io/docs/concepts/storage/storage-classes/#aws-ebs))
+#### Define Storage Class
+Copy/Paste the following commands into your Cloud9 Terminal. 
+```
+mkdir ~/environment/templates
+cd ~/environment/templates
+wget https://eksworkshop.com/statefulset/storageclass.files/mysql-storageclass.yml
+```
+Check the configuration of mysql-storageclass.yml file by following command.
+```
+cat ~/environment/templates/mysql-storageclass.yml
+kind: StorageClass
```

Comment:
If you want to provide example output, break it into a separate code box so it doesn't get copy/pasted with the cat command. There are examples elsewhere of wording like "The output should look something like this"